packages/now-cli/src/commands/dev/dev.ts

```diff
@@ -101,16 +99,9 @@ export default async function dev(
       cwd = join(cwd, project.rootDirectory);
     }
 
-    [environmentVars, systemEnvs] = await Promise.all([
-      getDecryptedEnvRecords(
-        output,
-        client,
-        project,
-        ProjectEnvTarget.Development
-      ),
-      project.autoExposeSystemEnvs
-        ? exposeSystemEnvs(output, client, project.id)
-        : undefined,
+    [{ envs: projectEnvs }, { systemEnvValues }] = await Promise.all([
+      getDecryptedEnvRecords(output, client, project.id),
+      getSystemEnvValues(output, client, project.id),
```

Comment:
Looks like the value is never used when `project.autoExposeSystemEnvs` is `false` so it'd be better to not retrieve values in that case.

Reply:
oh great point, fixed both in https://github.com/vercel/vercel/pull/5451/commits/15b7b9e52eca99a4fcc6751f52a1a44fe2a35da5 👍